.github/ISSUE_TEMPLATE/feature-enhancement.md

```diff
@@ -19,12 +19,9 @@ assignees: ''
 <!-- If you had a way to implement it in mind, describe it here, including any concept screenshots if possible -->
 
 
-## Possible Alternatives
-<!-- List any possible alternatives to your proposed implementation -->
-
-
 ## Additional Details
 <!-- Delete this section if not applicable. -->
+<!-- If alternate implementations were considered, add them here. -->
```

Comment:
Let's switch the order to make sure the second comment is read before the hasty decision is made to delete the section.